package/test_requirements.txt

```
@@ -1,13 +1,15 @@
 -r requirements.txt
-kedro[pandas,spark]>=0.16.0  
+kedro>=0.17.0  
```

```suggestion
kedro[pandas]>=0.17.0  
```
and then remove the `pandas` and `pyarrow` requirements below. Easiest to just rely on the version that kedro provides (assuming that works).

---

actually let me try that. it wasn't working initially because hdfs dependency was interfering with tables dependency on 3.9,3.10 (it's very weird that it's passing on kedro). but now that I have removed spark completely, the above might work.

---

@AntonyMilneQB  -- it doesn't work. It fails when installing pytables dependency. I will revert back to having it seperate for now.